docs/source/configuration/internalproxy.md

```diff
@@ -57,7 +57,7 @@ RAZZLE_API_PATH=http://localhost:8081/mysite yarn start
 ```
 
 !!! tip
-    To view the existing configuration, add console.log(config) to the `applyConfig` function. This dumps the existing config to your browser console.
+    To view the existing configuration, add console.log(config) to the `applyConfig` function. This dumps the existing config to your terminal console.
```

Comment:
This tip is meant for newbies, console.log does both (logs to terminal and to the browser console). I think it's easier to inspect the config in the browser.

Was there a problem seing the config in the browser console?

Reply:
I was looking into this with @pigeonflight and since it's in config, it's logged by the server